app/controllers/account/users_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Account::UsersController < ApplicationController
   layout 'dashboard'
+  before_action :is_admin?
```

Comment:
bad name of method

require_admin must BE
PLUS do we plan to use this method somewhere else? If yes - method must be in parent class and before_action just needs to be here